tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -177,6 +181,154 @@ TEST( OutputReportTabularTest, GetUnitConversion )
 
 }
 
+TEST( OutputReportTabularTest, AllocateLoadComponentArraysTest )
+{
+	ShowMessage( "Begin Test: OutputReportTabularTest, AllocateLoadComponentArraysTest" );
+
+	AllocateLoadComponentArraysDoAllocate = true;
```

Comment:
@mbadams5 It seemed like a good approach to me although I am not opposed to other approaches. What is the advantage of the change you are proposing?